art/api/models.py

```diff
@@ -1,3 +1,8 @@
 from django.db import models
 
-# Create your models here.
+class AssetType(models.Model):
```

Comment:
@HawiCaesar does  `pycode`  has a `two linespace rule` before a class? I think pep has.

Reply:
@jmnyarega pycodestyle is PEP8. Check out this [link](https://pypi.python.org/pypi/pycodestyle/1.8.0.dev0). @wanjiku-carol it looks like you have not installed the pre-commit hook to check for PEP8 violations.

Reply:
@HawiCaesar I believe there should be two spaces after imports.